tutorials/plot_object_source_estimate.py

```diff
@@ -0,0 +1,238 @@
+"""
+The :class:`SourceEstimate <mne.SourceEstimate>` data structure
+===============================================================
+
+
+Source estimates, commonly referred to as STC (Source Time Courses),
+are obtained from source localization methods.
+Source localization method solve the so-called 'inverse problem'.
+MNE provides different methods for solving it:
+dSPM, sLORETA, LCMV, MxNE etc.
+
+Source localization consists in projecting the EEG/MEG sensor data into
+a 3-dimensional 'source space' positioned in the individual subject's brain
+anatomy. Hence the data is transformed such that the recorded time series at
+each sensor location maps to time series at each spatial location of the
+'source space' where is defined our source estimates.
+
+An STC object contains the amplitudes of the sources over time.
+It only stores the amplitudes of activations but
+not the locations of the sources. To get access to the locations
+you need to have the :class:`source space <mne.SourceSpaces>`
+(often abbreviated `src`) used to compute the
+:class:`forward operator <mne.Forward>` (often abbreviated `fwd`).
+
+See :ref:`tut_forward` for more details on forward modeling, and
+:ref:`sphx_glr_auto_tutorials_plot_mne_dspm_source_localization.py`
+for an example of source localization with dSPM, sLORETA or eLORETA.
+
+Source estimates come in different forms:
+
+    - :class:`mne.SourceEstimate`: For cortically constrained source spaces.
+
+    - :class:`mne.VolSourceEstimate`: For volumetric source spaces
+
+    - :class:`mne.VectorSourceEstimate`: For cortically constrained source
+      spaces with vector-valued source activations (strength and orientation)
+
+    - :class:`mne.MixedSourceEstimate`: For source spaces formed of a
+      combination of cortically constrained and volumetric sources.
+
```

Comment:
osd was hitting me so: some of the list entries have a period and others don't + I would add something like "...cortically constrained source spaces (demonstrated here)"